src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/TaskManagementService.java

```diff
@@ -775,6 +776,16 @@ public void terminateTask(String taskId, TerminateInfo terminateInfo) {
         }
     }
 
+    @Transactional(propagation = Propagation.REQUIRES_NEW)
+    public TaskResource updateTaskIndex(String taskId) {
+        TaskResource taskResource = cftTaskDatabaseService.findByIdOnly(taskId).orElse(null);
+        if (taskResource != null) {
```

Comment:
Minor point, but if you're getting an Optional back then why not just use it to check if the result is present?  Usually better to avoid `null` where possible since it is ambiguous.

Reply:
Done